src/Affiliations.js

```diff
@@ -0,0 +1,9 @@
+export default class Affiliation {
+  constructor(affiliation) {
+    // TODO: name is readable but still might want to clean them up
+    // i.e "NewLokaSyndicate" can be "New Loka"" instead
```

Comment:
these are convertable from warframe-worldstate-data, i'll get the utilities for these exposed from warframe-worldstate-parser... might do it inside warframe-worldstate-data itself

Reply:
Alright I'll keep an eye out and add it soon as I see it